run/SU2_preCICE_CHT.py

```diff
@@ -57,7 +57,7 @@ def main():
   parser.add_option("-r", "--precice-reverse", action="store_true", dest="precice_reverse", help="Include flag to have SU2 write temperature, read heat flux", default=False)
   
   # Dimension
-  parser.add_option("-d", "--dimension", dest="nDim", help="Dimension of fluid domain", type="int", default=3)
+  parser.add_option("-d", "--dimension", dest="nDim", help="Dimension of fluid domain (2D/3D)", type="int", default=2)
```

Comment:
@j-signorelli FYI, I changed the default `nDim`, as the tutorials are anyway 2D. This might affect how you use the code in your research. Let me know if you think it is better we keep it 3D.